index.html

```diff
@@ -445,7 +445,11 @@
       await xrpackage.remove(p);
       p = null;
     }
-    p && p.setMatrix(new THREE.Matrix4().fromArray(value.matrix));
+    if(p) {
+      state[p.id].ignoreMatrixUpdate = true
```

Comment:
Used to prevent syncing state that was sent from a remote client.

Reply:
Not sure if there's a better way to do this. Modifying `setMatrix` in xrpackage.js or removing then reattaching the listener may be better.

Reply:
I would remove/re-add the event listener, since that is basically what this is trying to do, except with a hack.

Reply:
Will do